bittensor/wallet.py

```diff
@@ -115,6 +115,8 @@ def config(cls) -> "bittensor.config":
         Returns:
             bittensor.config: Config object.
         """
```

Comment:
@LogrisTheBard please check those 2 TODOs / Questions aswell:

Reply:
I don't have a good reason not to just pass in the config. This seems like something relevant enough to our task to refactor.